package/harvester-os/files/etc/cos/bootargs.cfg

```diff
@@ -2,7 +2,7 @@ set console_params="console=tty1"
 set kernel=/boot/vmlinuz
 set crash_kernel_params="crashkernel=219M,high crashkernel=72M,low"
 if [ -n "$recoverylabel" ]; then
-    set kernelcmd="$console_params root=live:LABEL=$recoverylabel rd.live.dir=/ rd.live.squashimg=$img panic=0 net.ifnames=1 rd.cos.oemtimeout=120"
+    set kernelcmd="$console_params root=live:LABEL=$recoverylabel rd.live.dir=/ rd.live.squashimg=$img panic=0 net.ifnames=1 rd.cos.oemtimeout=120 audit=1 audit_backlog_limit=8192"
```

Comment:
I guess we should add this in the else clause later. This only applies to recovery mode.

Reply:
oh. I will fix this.